.travis.yml

```diff
@@ -9,10 +9,6 @@ matrix:
     - go: tip
 
 install:
-  - rm -rf $GOPATH/src/gopkg.in/src-d
```

Comment:
Yes, that is the whole purpose of this PR - switch all things to master. See 4f42d95

If that is not the right thing to do in Golang, let's use this as disucssion\eduction point on how gopkg.in must be used.